README.md

```diff
@@ -0,0 +1,46 @@
+# flathub
+Pull requests for new applications to be added
+
+# org.gnome.Radio
+GNU Network Object Model Environment Radio 64.0 -- WWW.GNOMERADIO.ORG
+=====================================================================
+
+Radio is a Free Software program that allows you to easily
+locate and listen to Free Internet Radio stations by broadcasters on
+the Internet with the help of map and text search.
+
+Radio is developed on the GNOME desktop platform with GNOME
+Maps, libchamplain and geocode-lib and it requires gstreamer 1.0 for
+audio playback.
+
+You can view all the stations in src/gnome-radio.xml and enter city
+names in the GUI search input field in order to locate radio stations
+in the city using the text search with auto-completion or click on one
+of the city marker in the map to listen to a station.
+
+To launch the program, you can click on Activities in GNOME and enter
```

Comment:
this README will never be seen by users. So that you know what to expect.

Reply:
OK, I have added org.gnomeradio.Radio.appdata.xml with the following content:

<?xml version="1.0" encoding="UTF-8"?>
<application>
  <id type="desktop">org.gnome.Radio.desktop</id>
  <name>GNOME Radio 64 for GNOME 44</name>
  <summary>Listen to National Public Radio on GNOME 44</summary>
  <licence>CC0</licence>
  <description>
    <p>
      GNOME Radio 64 is a Free Software program that allows you easily
      listen to National Public Radio (NPR) broadcasts under GNOME 44.
    </p>
    <p>
      GNOME Radio 64 is developed on the GNOME 44 desktop platform and
      it requires GTK4, Clutter and GStreamer 1.0 for playback.
    </p>
    <p>    
    Enjoy National Public Radio (NPR) broadcasts under GNOME 44.
    </p>
  </description>
  <url type="homepage">https://wiki.gnome.org/Apps/Radio</url>
  <screenshots>
    <screenshot type="default">https://gitlab.gnome.org/ole/gnome-radio/-/blob/main/data/screenshots/gnome-radio-screenshot.png</screenshot>
  </screenshots>
  <updatecontact>ole@aamot.org</updatecontact>
</application>